optimum/neuron/utils/hub_neuronx_cache.py

```diff
@@ -189,12 +190,26 @@ def download_folder(self, folder_path: str, dst_path: str):
                 try:
                     # cached remotely
                     for repo_content in folder_info:
-                        # TODO: this works for `RepoFile` but not `RepoFolder`
-                        local_path = self.api.hf_hub_download(self.repo_id, repo_content.path)
-                        filename = Path(local_path).name
-                        dst_path = Path(dst_path)
-                        dst_path.mkdir(parents=True, exist_ok=True)
-                        os.symlink(local_path, dst_path / filename)
+                        if isinstance(repo_content, RepoFile):
```

Comment:
Wasn't it possible then to use snapshot_download instead ?

Reply:
Is snapshot_download usable for a subfolder under a repo instead of the whole repo? I tried it by setting `allow_patterns` as the hash key but there was no file matched...